uvicorn/loops/asyncio.py

```diff
@@ -1,11 +1,21 @@
 import asyncio
 import logging
 import sys
+from typing import Optional
 
 logger = logging.getLogger("uvicorn.error")
 
 
-def asyncio_setup(reload: bool = False) -> None:  # pragma: no cover
-    if sys.version_info >= (3, 8) and sys.platform == "win32" and reload:
-        logger.warning("The --reload flag should not be used in production on Windows.")
+def asyncio_setup(
+    reload: bool = False, workers: Optional[int] = None
+) -> None:  # pragma: no cover
+    if (
+        sys.version_info >= (3, 8)
+        and sys.platform == "win32"
+        and any([reload, workers])
+    ):
+        if reload:
+            logger.warning(
+                "The --reload flag should not be used in production on Windows."
+            )
```

Comment:
I'm probably missing something. But why do we have this warning here? 🤔 Why is it assuming this place is "windows production"?

Reply:
I don't think I'm missing anything... I think this warning is just wrong.

Reply:
I finally understand why this warning is here! It's because of this comment: https://github.com/encode/uvicorn/issues/1220#issuecomment-947920138

It has the wrong assumption, tho! It's not because we reach this code that it's running in production.